hoodie-common/src/main/java/com/uber/hoodie/common/HoodieRollbackStat.java

```diff
@@ -34,12 +33,18 @@
     private final List<String> successDeleteFiles;
     // Files that could not be deleted
     private final List<String> failedDeleteFiles;
+    private final Map<FileStatus, Long> commandBlocks;
 
     public HoodieRollbackStat(String partitionPath, List<String> successDeleteFiles,
-        List<String> failedDeleteFiles) {
+        List<String> failedDeleteFiles, Map<FileStatus, Long> commandBlocks) {
```

Comment:
Stats collected on Rollback for merge on read is basically keeping track of how many command blocks were written.